src/main/java/shapes/Demo.java

```diff
@@ -13,10 +15,14 @@ public static void main(String[] args) {
                 System.out.println("Area of circle = " + ((Circle) shape[i]).getArea());
             }
             if (shape[i] instanceof Rectangle) {
-                System.out.println("Perimeter of rectangle = "+((Rectangle) shape[i]).getPerimeter());
+                System.out.println("Perimeter of rectangle = " + ((Rectangle) shape[i]).getPerimeter());
             }
+            if (shape[i].isInside(x, y)==true) {
+                System.out.println("Point (" + x + "," + y + ") is inside the shape.");
+            } else System.out.println("Point (" + x + "," + y + ") is outside the shape");
```

Comment:
Please don't ignore `{}` brackets even for one line blocks.
It's common good code style.